_includes/article.html

```diff
@@ -35,43 +31,41 @@ <h2 class="entry-title">{% if site.titlecase %}{{ page.title | titlecase }}{% el
       <div class="entry-content">{{ content }}</div>
     {% endif %}
 
-    {% if page.single == true %}
-      <p class="meta-author">
-        {% include post/author.html %}
-      </p>
-      <p class="meta-categories">
-        Categories: {% include post/categories.html %}
-      </p>
-      {% if page.series %}
-       <p class="meta-series" style="padding-bottom:0;">Part of a series: <strong>{{ page.series | series_link }}</strong></p>
-       <ul>
-         {% for post in site.posts %}
-          {% if post.series == page.series %}
-          <li>{% include archive_post.html %}</li>
-          {% endif %}
-        {% endfor %}
-
-       </ul>
-      {% endif %}
+    <p class="meta-author">
+      {% include post/author.html %}
+    </p>
+    <p class="meta-categories">
+      Categories: {% include post/categories.html %}
+    </p>
+    {% if page.series %}
+      <p class="meta-series" style="padding-bottom:0;">Part of a series: <strong>{{ page.series | series_link }}</strong></p>
+      <ul>
+        {% for post in site.posts %}
+        {% if post.series == page.series %}
+        <li>{% include archive_post.html %}</li>
+        {% endif %}
+      {% endfor %}
 
-      {% if site.disqus_short_name and page.comments == true %}
-        <hr/>
-        <section class="comments">
-          <h1>Comments</h1>
-          <div id="disqus_thread" aria-live="polite">{% include post/disqus_thread.html %}</div>
-        </section>
-        <hr/>
-      {% endif %}
+      </ul>
+    {% endif %}
 
-      <p class="meta-paginate">
-        {% if page.previous.url %}
-          <a class="previous" href="{{page.previous.url}}" title="Previous Post: {{page.previous.title}}">&laquo; {{page.previous.title}}</a>
-        {% endif %}
-        {% if page.next.url %}
-          <a class="next" href="{{page.next.url}}" title="next Post: {{page.next.title}}">{{page.next.title}} &raquo;</a>
-        {% endif %}
-      </p>
+    {% if site.disqus_short_name and page.comments == true %}
+      <hr/>
+      <section class="comments">
+        <h1>Comments</h1>
+        <div id="disqus_thread" aria-live="polite">{% include post/disqus_thread.html %}</div>
+      </section>
+      <hr/>
     {% endif %}
+
+    <p class="meta-paginate">
+      {% if page.previous.url %}
+        <a class="previous" href="{{page.previous.url}}" title="Previous Post: {{page.previous.title}}">&laquo; {{page.previous.title}}</a>
+      {% endif %}
+      {% if page.next.url %}
+        <a class="next" href="{{page.next.url}}" title="next Post: {{page.next.title}}">{{page.next.title}} &raquo;</a>
+      {% endif %}
+    </p>
```

Comment:
For some reason the `single: true` YAML front-matter stopped being passed through layouts, so this include, which was previously used for both full- and abbreviated- (list) versions is now only used for the full article experience. The `article_list_item` include below handles the abbreviated experience (e.g., for the site's home page).